public/index.html

```diff
@@ -29,13 +29,7 @@ <h1 class="header__text">СОБЫТИЯ</h1>
     <script>
       if ('serviceWorker' in navigator) {
         window.addEventListener('load', function() {
-        navigator.serviceWorker.register('/sw.js').then(function(registration) {
-            // Registration was successful
-            console.log('ServiceWorker registration successful with scope: ', registration.scope);
-            }, function(err) {
-            // registration failed :(
-            console.log('ServiceWorker registration failed: ', err);
-          });
+          navigator.serviceWorker.register('sprava_web_app/service-worker.js');
```

Comment:
maybe we need to provide comment that we need that beause of github pages ?

Reply:
I fix wrong way to file, but unfortunately delete comments. I fix this